src/lib.rs

```diff
@@ -325,14 +325,44 @@ where
     }
 }
 
+impl<K, V, S: BuildHasher> FlurryHashMap<K, V, S> {
+    /// Creates a new, empty map with the default initial table size (16).
+    /// It uses the given hasher, rather than the default `RandomState`.
+    pub fn new_with_hasher(hasher: S) -> Self {
+        Self {
```

Comment:
We probably want `fn new` to call this method rather than have both of them separately construct a `Self`.